tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -0,0 +1,126 @@
+#
+# Common widget google-rtc-aec
+#
+# A generic google-rtc-aec widget. All attributes defined herein are namespaced
+# by alsatplg to "Object.Widget.google-rtc-aec.N.attribute_name"
+#
+# Usage: this component can be used by instantiating it in the parent object. i.e.
+#
+# 	Object.Widget.google-rtc-aec."N" {
+# 		index		1
+# 		type		"effect"
```

Comment:
just curious why `type` is needed at the top level? Isn't it implied by the class?

Reply:
Yes I get that by why is it recommended at the pipeline level, isn't every aec a effect? That doesn't explain why

Reply:
why do we need to set this in pipeline level ? It should be in widget level. Here is at widget level

Reply:
I am not sure I understand the levels...

Reply:
I am also not sure I understand the levels...  Here type is a example in comments for example